indigo_api/models.py

```diff
@@ -105,6 +105,15 @@ def for_code(cls, code):
         return cls.objects.get(country__pk=code.upper())
 
 
+@receiver(signals.post_save, sender=Country)
+def post_save_country(sender, instance, **kwargs):
+    """ When a country is saved, make sure a PlaceSettings exists for it.
+    """
+    if not instance.place_settings.all():
```

Comment:
I think you need to have a filter on `locality=None`, otherwise place settings for a locality will show up in this list.